proto/viam/service/vision/v1/vision.proto

```diff
@@ -160,3 +168,16 @@ message GetObjectPointCloudsResponse {
   // List of objects in the scene
   repeated common.v1.PointCloudObject objects = 2;
 }
+
+message GetPropertiesRequest {
+  // name of the vision service
+  string name = 1;
+  // Additional arguments to the method
+  google.protobuf.Struct extra = 99;
+}
+
+message GetPropertiesResponse {
+  bool classifications_supported = 1;
+  bool detections_supported = 2;
+  bool object_point_clouds_supported = 3;
```

Comment:
Would be nice to have docstrings here, even though the docstrings would be super self explanatory. It's just very useful with auto-generated documentation in the SDKs